bin/MorphingMSSMvsSM.cpp

```diff
@@ -538,88 +613,106 @@ int main(int argc, char **argv) {
         --catit;
       }
     }
-    // std::cout << "[INFO] Using the following categories:" << std::endl;
-    // std::cout << "   sm_and_btag_cats:" << std::endl;
-    // for (const auto i: sm_and_btag_cats)
-    //   std::cout << "      " << i.first << ' ' << i.second << std::endl;
-    // std::cout  << std::endl;
-    // std::cout << "    mssm_cats:" << std::endl;
-    // for (const auto i: mssm_cats)
-    //   std::cout << "      " << i.first << ' ' << i.second << std::endl;
-    // std::cout  << std::endl;
-    // std::cout << "    mssm_btag_cats:" << std::endl;
-    // for (const auto i: mssm_btag_cats)
-    //   std::cout << "      " << i.first << ' ' << i.second << std::endl;
-    // std::cout  << std::endl;
+
+    std::cout << "[INFO] Using the following categories:" << std::endl;
+    std::cout << "   sm_and_btag_cats:" << std::endl;
+    for (const auto i: sm_and_btag_cats)
+      std::cout << "      " << i.first << ' ' << i.second << std::endl;
+    std::cout  << std::endl;
+    std::cout << "    mssm_cats:" << std::endl;
+    for (const auto i: mssm_cats)
+      std::cout << "      " << i.first << ' ' << i.second << std::endl;
+    std::cout  << std::endl;
+    std::cout << "    mssm_btag_cats:" << std::endl;
+    for (const auto i: mssm_btag_cats)
+      std::cout << "      " << i.first << ' ' << i.second << std::endl;
+    std::cout  << std::endl;
+    std::cout << "    sm_signal_cat:" << std::endl;
+    for (const auto i: sm_signal_cat)
+      std::cout << "      " << i.first << ' ' << i.second << std::endl;
+    std::cout  << std::endl;
+
     cb.AddObservations({"*"}, {"htt"}, {era_tag}, {chn}, cats[chn]);
     cb.AddProcesses({"*"}, {"htt"}, {era_tag}, {chn}, bkg_procs[chn], cats[chn], false);
-    // currently possible analysis:
-    // 1. sm
-    // 2. mssm
-    // 3. mssm_classic
-    // 3. mssm_vs_sm_h125
-    // 4. mssm_vs_sm
-
 
     if(analysis == "sm"){
       cb.AddProcesses({""}, {"htt"}, {era_tag}, {chn}, main_sm_signals, cats[chn], true);
     }
-    else if(analysis == "mssm"){
+    else if(analysis == "bsm-model-indep"){
       // filter masses above treshold:
-      std::vector<std::string> masses_SM_bbH;
-      std::vector<std::string> masses_SM_ggH;
-      int bbH_threshold = SM_thresholds_bbH[era][chn];
-      int ggH_threshold = SM_thresholds_ggH[era][chn];
+      // std::vector<std::string> masses_SM_bbH;
```

Comment:
If it is deprecated I can just remove that. @harrypuuter may comment on that.

Reply:
I used this in the past to switch between using susy and sm weighted depending on a threshold for the model independent limits with sm, but since we dont need that and wont use it, this can be removed

Reply:
removed